requirements.txt

```diff
@@ -46,7 +46,7 @@ requests==2.11.1
 rfc3986==0.4.1            # via jsontableschema
 six==1.10.0               # via datapackage, goodtables, jsonlines, linear-tsv, python-dateutil, tabulator
 sqlalchemy==1.1.4
-tabulator==0.13.0
+tabulator==0.14.0
```

Comment:
This should be added to requirements.in, and then run `make requirements` to update this one